ts/global.d.ts

```diff
@@ -11,6 +11,7 @@ interface Window {
   mnemonic: any;
   clipboard: any;
   attemptConnection: any;
+  setZoomFactor: any;
```

Comment:
this doesn't seem to be used in any typescript code so we can also remove this.

Reply:
removed